src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -94,6 +94,12 @@ protected function hasColorSupport()
         if (function_exists('stream_isatty') && !@stream_isatty($this->stream)) {
             return false;
         }
+
+        $colorEnv = strtolower(getenv('ANSI', true) ?: getenv('ANSI'));
+        if ('' !== $colorEnv) {
+            return in_array($color, array('true', 'y', 'yes', '1'));
```

Comment:
Could use `filter_var` with [`FILTER_VALIDATE_BOOLEAN` flag](http://php.net/manual/en/filter.filters.validate.php) if we want to support many more standard values.